modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerSslTest.java

```diff
@@ -41,6 +41,15 @@
     /** */
     private volatile String[] cipherSuites;
 
+    /** {@inheritDoc} */
+    @Override protected void afterTest() throws Exception {
```

Comment:
Looks like we need one more abstract class for all inherritors of `GridCommandHandlerAbstractTest`  except `GridCommandHandlerClusterByClassTest` with common beforeTest()/afterTest() logic